backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -11,16 +12,29 @@ const users = pgTable(
   {
     id: serial('id').primaryKey(),
     phone: varchar('phone').notNull(),
+    email: varchar('email'),
+    firstName: varchar('first_name'),
+    lastName: varchar('last_name'),
```

Comment:
these fields should be not nullable ?

Reply:
I thought user need a phone and password only for creating account and the rest info is additional. Ok, I had a wrong logic.